functionnal_tests/cypress/integration/workspace/create_workspace_spec.js

```diff
@@ -21,7 +21,7 @@ describe('Workspace', () => {
       .get('.newSpace__button .btn').last().should('be.visible').click()
   }
 
-  it('should create a new space while no others are created', () => {
+  it.skip('should create a new space while no others are created', () => {
```

Comment:
Do you know if an issue exist about stabilization of this test ? (make sure other dev know this problem)

Reply:
Yup #5091

Reply:
ok